idunn/instant_answer/patterns_to_ignore.txt

```diff
@@ -23,3 +23,4 @@ lieu dit
 localisation
 o(ù|u) (se|ce) (trouve|situe)( (le|la))?
 where is
+booking$
```

Comment:
Why did we choose to match only at the end of the string?

Reply:
To reduce the risk of false positives, when the keyword may be helpful to recognize and classify other types of intention.  
When associated to a hotel name, "booking" appears almost always at the end of the query (statistically).